### ISSUE TITLE
tf.weighted_cross_entropy_with_logits expects logits NOT probability

### ISSUE THREAD

**Issue by lc82111** [harsimrat-eyeem/holy-edge#1]
https://github.com/harsimrat-eyeem/holy-edge/blob/98cf525de5ed787e58b521be2e60a03b9da4826c/hed/losses.py#L21

weighted_cross_entropy_with_logits(
    targets,
    logits,
    pos_weight,
    name=None
)
expects logits NOT probability: sigmoid(logits)

**Comment by lc82111** [comment]
sorry, my fault.
